hw/mcu/nxp/kinetis/src/hal_hw_id.c

```diff
@@ -22,26 +22,27 @@
 
 #include <hal/hal_bsp.h>
 
-#include "MK64F12.h"
+#include "fsl_sim.h"
 
 #ifndef min
 #define min(a, b) ((a)<(b)?(a):(b))
 #endif
 
-#define MK64F12_HW_ID_LEN     7
+#define MK8xF_HW_ID_LEN sizeof(sim_uid_t)
```

Comment:
Should be called `KINETIS_HW_ID_LEN` or some similar family name.

Reply:
Addressed.